lib/plugins/aws/info/index.js

```diff
@@ -184,40 +198,203 @@ ${chalk.yellow('region:')} ${info.region}`;
               path = event.http.split(' ')[1];
             }
             path = path !== '/' ? `/${path.split('/').filter(p => p !== '').join('/')}` : '';
-            endpointsMessage = endpointsMessage.concat(`\n  ${method} - ${info.endpoint}${path}`);
+            endpointsMessage += `\n  ${method} - ${info.endpoint}${path}`;
           }
         });
       });
     } else {
-      endpointsMessage = endpointsMessage.concat(`\n  None`);
+      endpointsMessage += `\n  None`;
     }
 
-    message = message.concat(endpointsMessage);
+    message += endpointsMessage;
 
-    // Display Functions
+    // Display Functions and their metrics
     let functionsMessage = `\n${chalk.yellow('functions:')}`;
 
     if (info.functions && info.functions.length > 0) {
       info.functions.forEach((f) => {
-        functionsMessage = functionsMessage.concat(`\n  ${f.name}: ${f.arn}`);
+        functionsMessage += `\n  ${f.name}:\n`;
+        functionsMessage += `    ${chalk.yellow('arn:')} ${f.arn}\n`;
+
+        // display metrics
+        // NOTE: we only get one datapoint because the metrics are from the last 24h
+        functionsMessage += `    ${chalk.yellow('metrics (last 24h):')}\n`;
+        if (f.metrics.length && f.metrics.length > 0) {
+          f.metrics.forEach((metric) => {
+            if (metric.Label === 'Invocations') {
+              functionsMessage += `      ${chalk.yellow('invocations:')}`;
+              if (metric.Datapoints.length) {
+                functionsMessage += ` ${metric.Datapoints[0].Sum}x`;
```

Comment:
imo let's ditch the `x` ... so it'd be `invocations: 2`

Also, is there a reason why it's scoped only to the last 24h?!